### DISCUSSION TITLE
Code trace is not available for this event after build

### DISCUSSION THREAD

**Discussion by icyrogue** [day8/re-frame#767] [Q&A]
Hello there.

I wanted to try re-frame, so I built a new project using re-frame lein template and 10x following steps on template's github repo  

When I run ` npm install ` and ` npm run watch ` using ` node v16.14.2 (npm v8.5.0) ` 
<details>
<summary> I get: </summary>

```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```

</details>

After build, I go to localhost and get error in 10x window
<details>
<summary>Screenshot:</summary>
<img width="446" alt="Screenshot 2022-10-30 at 5 03 28 PM" src="https://user-images.githubusercontent.com/89037640/198882864-0b314ee9-fec6-45aa-a91e-506aa1eb13d0.png">

</details>

In my editor (neovim) I can connect to REPL just fine, but in ``` core.cljs ``` I get ``` symbol _ _ is unresolved  ``` in ```  fn-traced [_ _] ```

<details>
<summary>Screenshot:</summary>
<img width="453" alt="Screenshot 2022-10-30 at 5 10 02 PM" src="https://user-images.githubusercontent.com/89037640/198883211-1be6f154-c49a-4ed7-850c-103f6e85e9f5.png">
</details>

Can someone point me in the right direction? Why is this happening?  Thanks for any help in advance

**Comment by kimo-k** [comment]
Hey, sorry for the late reply. I'm just noticing there are a lot of discussions here.

SLF4J messages are nominal, they don't indicate any problem AFAIK.

`symbol _ _ is unresolved` - is that the literal error message? If so, maybe that's a clue. `_ _` should be two symbols, not one. I would check if the space between the underscores is an actual space. Sometimes you can end up with an alternate whitespace char, e.g. https://www.compart.com/en/unicode/category/Zs. They have been known to confuse the reader. I can't think of any other reason it would read `_ _` as a single symbol.
